.github/workflows/all-green.yml

```
@@ -14,15 +18,34 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       checks: read
-      contents: read
+      statuses: read
     steps:
-      - uses: wechuli/allcheckspassed@e22f45a4f25f4cf821d1273705ac233355400db1
+      - uses: DataDog/ensure-ci-success@f40e6ffd8e60280d478b9b92209aaa30d3d56895
         with:
-          delay: 10  # wait for this delay before starting
-          polling_interval: 1 # after a failure, how long do it wait before checking again
-          retries: 60  # how many retries before stopping 
-          # checks_exclude: ''  # comma separated list of checks to exclude, nothing right now
+          initial-delay-seconds: 10  # wait for this delay before starting
+          max-retries: 60  # how many retries before stopping 
+          ignored-name-patterns: |
+            consolidated-pipeline
+            throughput
+            dd-gitlab/run-benchmarks
+            .github/dependabot.yml
+            dd-gitlab/default-pipeline
+            benchmarks
+            profiler_execution_benchmarks
+            dd-gitlab/download-serverless-artifacts
+            dd-gitlab/download-single-step-artifacts
+            throughput_profiler
```

Throughput have been deleted.

```suggestion
            dd-gitlab/run-benchmarks
            .github/dependabot.yml
            dd-gitlab/default-pipeline
            benchmarks
            profiler_execution_benchmarks
            dd-gitlab/download-serverless-artifacts
            dd-gitlab/download-single-step-artifacts
```